.editorconfig

```diff
@@ -10,3 +10,6 @@ end_of_line = lf
 
 [Makefile]
 indent_style = tab
+
+[{*.yaml, *.yml}]
+indent_size = 2
```

Comment:
any reason for this? we do pretty much the opposite in crawlee and others, why not use the same idents across all languages?

Reply:
We have it like this in all the platform repos, plus it's the default indent used in the GitHub documentation: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

And we had it like this in all the other workflows apart from the docs one, so it was easier to unify it that way.